src/assets/SvgWrapper.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const SvgWrapper = ({ width = 24, height = 24, viewBox = '0 0 24 24', style={}, children }) => {
+  return (
+    <svg
+      style={style}
+      xmlns="http://www.w3.org/2000/svg"
+      width={width}
+      height={height}
+      viewBox={viewBox}
+    >{children}</svg>
+  );
+}
+
+export default SvgWrapper;
```

Comment:
remove semi-colons from all changed files because we are using the no-semi-colon approach.

Also, indent code